app/demoservice/settings.py

```diff
@@ -163,9 +163,8 @@
     host=RABBITMQ_HOST,
     port=RABBITMQ_PORT,
 )
-if DEBUG:
-    # In debug, run synchronously
-    CELERY_TASK_ALWAYS_EAGER = True
+
+CELERY_TASK_ALWAYS_EAGER = DEBUG or os.environ.get("CELERY_TASKS_ASYNC", False)
```

Comment:
Why change this? What is the situation where you might need to run CELERY_TASKS_ASYNC without DEBUG? I don't mind it being set explicitly, but I feel it should be one or the other - as in, you should either leave it as `CELERY_TASK_ALWAYS_EAGER = DEBUG`, or you should make it `CELERY_TASK_ALWAYS_EAGER = os.environ.get("CELERY_TASKS_ASYNC", False)` (since you have the power to set all your local development environment variables anyway through the `.env-dev` file).

Reply:
This setting was mostly useful to run the demoservice using manage.py  as it tells celery to run the tasks in the same process as the runserver. I might follow the suggestion and update the README saying that if you want to use the runserver it needs to be started with `DJANGO_DEBUG=True CELERY_TASK_ALWAYS_EAGER=True python3 ./app/manage.py runserver`